clusterloader2/pkg/state/tuningsets.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package state
```

Comment:
My feeling is that we can get rid of this whole file by just changing the api to make TuningSets a map (with key as the tuningset name) instead of an array. Then once you read the config, you'll have the struct readily available.

cc @wojtek-t

Reply:
Actually thinking more about it - I feel like tuningsets aren't exactly a state (nothing is changing with them). They are more or less constants, so I feel even stronger about this change.

Reply:
I removed tuningsets.go - we are not using tuning sets at this moment.
I removed state.go - it would only contain namespacesState so there was not point of keeping it.